composer.json

```diff
@@ -5,7 +5,8 @@
     "require": {
         "php": "^7.1",
         "behat/behat": "^3.3",
-        "psr/container": "^1.0"
+        "psr/container": "^1.0",
+        "symfony/dependency-injection": "^3.3"
```

Comment:
You should be able to fix the tags instead, in the services.yml - I'll look at that as a separate change after this is merged though